### ISSUE TITLE
project run stopped working

### ISSUE THREAD

**Issue by matjaz** [athombv/homey#225]
**What did you try to achieve?**
`athom project --run`

**What was the result?**
```
$ athom project --run
Archiving...
Uploading to...
Error: socket hang up
```
Blue ledring starts to spin, but then in like 30s it stops and in some more time "socket hang up" error is displayed.

How can I debug this? I guess I need @athombv support.

**What did you expect as result?**
app ran

**Any other remarks**
I did a full data reset. Issue stays.

**What is your Homey version?**
0.8.17

**Comment by matjaz** [comment]
Hope you guys have some logging, which would help us to troubleshoot issues like that ourselves.

**Comment by WeeJeWel** [comment]
Try `athom login` again. I think your Homey's local ip changed.

**Comment by matjaz** [comment]
I already did that. Ip is the same. App starts to upload (blue ledring).
Now I noticed app appears in apps settings page, but it's not running.
`Debugging...` text is not displayed.. as it was before.

**Comment by matjaz** [comment]
I just tried with fresh app `athom project --create`. same result. spinning blue ledring, ledring off, and then few seconds after socket timeout error.

**Comment by matjaz** [comment]
I did a data reset **after** issue appeared.

**Comment by matjaz** [comment]
Looking into athom-cli it looks like homey does not respond to POST `/api/manager/devkit/`.

Interestingly I can install only apps from app store, that were already previously installed (com.icndb.api).
But new apps installed for the first time (com.8ball, it.diederik.wol) I get `504 GATEWAY_TIMEOUT`
New failed app I can see in settings/apps, but the svg icon is missing (404).

**Comment by WeeJeWel** [comment]
Fixed in 0.8.19
